boards/DCM/Inc/App/states/App_AllStates.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "App_SharedStateMachine.h"
+
+/**
+ * On-tick function for every state in the given state machine
+ * @param state_machine The state machine to run on-tick function for
+ */
+void App_AllStatesRunOnTick(struct StateMachine *state_machine);
```

Comment:
@garethellis0 I didn't want to write this code snippet 3 times (once for each state)
```
    struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
    struct DcmCanTxInterface *can_tx      = App_DcmWorld_GetCanTx(world);
    struct DcmCanRxInterface *can_rx      = App_DcmWorld_GetCanRx(world);
    struct BrakeLight *       brake_light = App_DcmWorld_GetBrakeLight(world);

    bool is_brake_actuated =
        App_CanRx_FSM_BRAKE_GetSignal_BRAKE_IS_ACTUATED(can_rx);
    bool is_regen_active =
        App_CanTx_GetPeriodicSignal_TORQUE_REQUEST(can_tx) < 0.0f;

    App_BrakeLight_SetLightStatus(
        brake_light, is_brake_actuated, is_regen_active);
```

It didn't really belong in `SetPeriodicCanSignals.c` too, so I created this file as a way to share functions that should run in every state. What do you think?